specs/zkevm-prover.md

```diff
@@ -130,24 +55,32 @@ interface ZKVerifier {
 }
 ```
 
-## Implementation
+## Prover as an RPC Server
 
-See [kroma-prover](https://github.com/kroma-network/kroma-prover) for details.
+[kroma-prover](https://github.com/kroma-network/kroma-prover) is implemented as a gRPC server that generates zkevm
+proofs for the requested height block. It can be utilized as a local component on the Challenger's machine
```

Comment:
```suggestion
[kroma-prover](https://github.com/kroma-network/kroma-prover) is implemented as a gRPC server that generates a zkevm
proof for the requested height block. It can be utilized as a local component on the challenger's machine
```

Reply:
applied